Taarafo.Core/Brokers/Storages/StorageBroker.PostImpressions.cs

```diff
@@ -27,6 +28,19 @@ public async ValueTask<PostImpression> InsertPostImpressionAsync(PostImpression
             return postImpressionEntityEntry.Entity;
         }
 
+        public async ValueTask<PostImpression> SelectPostImpressionByIdsAsync(
+            Guid PostId,
+            Guid ProfileId)
+        {
+            using var broker =
+                new StorageBroker(this.configuration);
+
+            broker.ChangeTracker.QueryTrackingBehavior =
+                QueryTrackingBehavior.NoTracking;
```

Comment:
You don't need that anymore.

Reply:
I removed these two lines from the file.